### ISSUE TITLE
Any plans on publishing this?

### ISSUE THREAD

**Issue by andreasknoepfle** [dev-rsilver/plugin-react-static-images#5]
Hello @dev-rsilver 👋,

I noticed that the package is not yet available through npm package repos 
https://www.npmjs.com/package/plugin-react-static-images
Currently the only way I saw using this was: 

1. adding it to package.json with the github url.
2. adding a postinstall `yarn build`, so the `dist` folder gets built.

Do you have any plans on publishing it to the official repos, so one can use it directly? 
Or do you publish this already somewhere else? 

Cheers Andi

**Comment by dev-rsilver** [comment]
You're right, the plugin isn't currently published. It was meant to be a starting point for such a plugin for React Static and I was looking for feedback from the React Static community. Many of the other plugins for React Static are consolidated into the React Static repository itself, which factored into my decision not to publish it.

Unfortunately, I started work on the plugin shortly before React Static's lead maintainer departed that project.

The plugin has sort of been in limbo, since I haven't had a clear sense of whether React Static will continue to gain traction or if most of its users will migrate elsewhere.

I posted instructions in the React Static issues related to how to test the plugin by using a Github url and building the dist folder, which I'm guessing you found. That method of installation was really only meant for obtaining feedback from the previous maintainer and community since the repo could change at any time.

If you're using the plugin in a project, you might consider at the very least forking the repo, so that you have a stable, long-term copy.
